app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -199,6 +199,21 @@ open class DefaultToolbarMenu(
     } ?: false
     // End of predicates //
 
+    val installToHomescreen = BrowserMenuHighlightableItem(
```

Comment:
Just a thought, I am starting to think it would be worthwhile when we refactor all of these to add a comment for each of these items so we can easily decipher what they are without looking too hard at the code. The comment could just be the literal label or some variant for the menu item that we would see in the UI.

For instance: 
```suggestion
    // Install
    val installToHomescreen = BrowserMenuHighlightableItem(
```

Reply:
I have a PR in progress right now to refactor all of this 👍

Reply:
https://github.com/mozilla-mobile/fenix/pull/18831